saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/registration/RelyingPartyRegistration.java

```diff
@@ -81,6 +81,8 @@
 
 	private final Saml2MessageBinding assertionConsumerServiceBinding;
 
+	private final String nameIDFormat;
```

Comment:
For consistency, will you please lowercase acronyms like so: `nameIdFormat`. Please do the same for the associated getter and setter.